cmd/lakefs/cmd/run.go

```diff
@@ -531,6 +536,17 @@ func enableKVParamsMetrics(p params.Config) params.Config {
 	return p
 }
 
+func initOTFService() *diff.Service {
+	pluginsManager := plugins.NewManager[diff.Differ]()
+	pid := plugins.PluginIdentity{Version: 0, ExecutableLocation: "~/.lakefs/plugins/delta"}
```

Comment:
We documented `ExecutableLocation` as a full path.  So you'll need to call some function to expand the "~" here.  Also, this must be configurable: it may be hard / inconvenient to write the home directory in some setups.  In a containerized setting I would probably prefer to mount it into someplace such as /etc/lakefs/plugins/delta.

Reply:
This is configurable. Haven't been implemented obviously...